lib/cli.rb

```diff
@@ -53,6 +54,9 @@ def self.exit_on_failure?
     desc 'upgrade SUBCOMMAND ...ARGS', 'Various version upgrade utilities'
     subcommand 'upgrade', Mastodon::UpgradeCLI
 
+    desc 'email_domains SUBCOMMAND ...ARGS', 'Manage E-mail domains'
+    subcommand 'email_domains', Mastodon::EmailDomainsCLI
```

Comment:
I believe you can/should use `-` instead of `_` in CLI.

Also, while the command is called `email-domains`, the subcommands in it are only for listing and adding blocks, so it is a misnomer. I see the following options:

- Rename `list` to `list-blocks`
- Make `list` return *all* e-mail domains and add a `--blocked` option to list only blocked domains (expensive and probably useless, though). The command could also return nothing unless `--blocked` is passed (placeholder)
- Rename `email-domains` to `email-domain-blocks` and rename `block` to `add` and `unblock` to `remove`
- Integrate the subcommands with the `domains` command instead

Reply:
I will change email_domains to email-domain-blocks.
But already `preview_cards` is exists.

Reply:
I obey to your comment, but I figure out…

- Thor requires to use '_' instead of '-' in `subcommand` definition. when i use '-'  in the definition, the subcommand is not recognized. 
- Thor recognize both of '_' and '-' when command was invoked.
- Thor use '_'  in subcommand help.
- Thor use capitalization in subcommand `help`.

```
$ docker-compose run --rm web tootctl email-domain-blocks
Commands:
  tootctl email_domain_blocks add [DOMAIN...]     # add E-mail domain blocks
  tootctl email_domain_blocks help [COMMAND]      # Describe subcommands or one specific subcommand <=== THIS LINE
  tootctl email_domain_blocks list                # list E-mail domain blocks
  tootctl email_domain_blocks remove [DOMAIN...]  # remove E-mail domain blocks
```